PoGo.NecroBot.Logic/Tasks/FarmPokestopsTask.cs

```diff
@@ -208,5 +217,14 @@ public static class FarmPokestopsTask
 
             return pokeStops.ToList();
         }
+
+        private static async Task DownloadProfile(ISession session)
```

Comment:
Personally, I think this should have the comment on why it was copied from LoginState and made static, since otherwise it purely looks like it is duplicated (aside from the fact that it is static here and not in LoginState)